### ISSUE TITLE
Show hyperparameters used on each client's training

### ISSUE THREAD

**Issue by eyp** [eyp/federated-learning-network#3]
When the server requests a client to do a training, it sends the hyperparameters of the Federated Learning training. 
Thus is:

num of epochs, learning rate & batch size.

Show these hyper parameters along the client URL (see #2) in the same line.

To develop this the server will need a map to store the clients registered and their data associated (in this case the hyperparams).
